app/lib/mock_router_api.rb

```diff
@@ -0,0 +1,35 @@
+# When DISABLE_ROUTER_API is set to true, the application will use this
+# class instead of GdsApi::Router
+# This is to allow dual-running of both MongoDB and PostgreSQL Content Stores
+# for a transition period during the migration to PostgreSQL
+class MockRouterApi
```

Comment:
I wonder if you might want to simplify this class to be just have `def method_missing`  and log that, then you'd not need to be considering the actual API of the adapter and could have much fewer tests

Reply:
done